Makefile

```diff
@@ -27,3 +27,6 @@ watch_and_start:
 
 docker: test
 	docker build -t sdcplatform/response-operations-ui:latest .
+
+load-templates:
```

Comment:
Is this essential?  If so, then it should be added to make build by adding it to the line above
`build: load-templates`

Reply:
I've added this to make build.